app/controllers/journeys/capitalallowances/tailoring/ClaimCapitalAllowancesController.scala

```diff
@@ -53,14 +55,29 @@ class ClaimCapitalAllowancesController @Inject() (override val messagesApi: Mess
 
   private val page = ClaimCapitalAllowancesPage
 
+  private val netAmount          = BigDecimal(12345.67)
```

Comment:
These values are now fetchable and don't need to be hardcoded. Is that in scope for this ticket? The ticket is unclear but I can't find another ticket in the backlog to update this with real values. Might be something to clarify with Maishara